### DISCUSSION TITLE
Accordion - disable opacity on open/close

### DISCUSSION THREAD

**Discussion by rossthedevigner** [chakra-ui/chakra-ui#4618] [Q&A]
Is there documentation around how to disable the opacity animation when using the `<Accordion />` component? Thanks!

**Comment by jesstelford** [comment]
The closest I've found is using the `reduceMotion` prop on the `<Accordion>` component:

```jsx
<Accordion reduceMotion>
  ...
</Accordion>
```

Unfortunately this also removes the height animation, and the button animation.

EDIT:

Under the hood, Chakra uses `<Collapse>` which does accept a boolean prop `animateOpacity`, but unfortunately the `<AccordionPanel>` doesn't allow setting the value:

https://github.com/chakra-ui/chakra-ui/blob/main/packages/accordion/src/accordion.tsx#L228-L230

**Reply by anubra266** [reply]
You can open an issue for this feature, you can tag me

**Comment by PCPbiscuit** [comment]
Has this been resolved? Couldn't find a solution without using `reduceMotion`. 

update: 

Passing `motionProps={{ animateOpacity: false }}` to AccordionPanel solved the problem
